src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -556,6 +572,18 @@ internal bool IsNetStandardFeatureFlagEnabled()
             return headerTags;
         }
 
+        internal string[] ParseUrlStringArray(string commaSeparatedValues)
```

Comment:
This is a generic "split and trim" method. No urls, no parsing. Can we rename it to match? `SplitAndTrim()`?

Reply:
Done